pyproject.toml

```diff
@@ -3,11 +3,11 @@ requires = ["flit_core >=2,<4", "requests >= 2.23.0"]
 build-backend = "flit_core.buildapi"
 
 [tool.flit.metadata]
-module = "msgraphcore"
+module = "msgraph"
 author = "Microsoft"
 author-email = "graphtooling@microsoft.com"
```

Comment:
make the author address `graphtooling+python@microsoft.com`. This can help with tracking and email message rules.